src/Data/Functor/Foldable.hs

```diff
@@ -791,15 +792,17 @@ transverse n = cata (fmap embed . n)
 -- :}
 -- "FoObAr"
 --
--- We can implement `zipWith`
+-- We can implement a variant of `zipWith`
+--
+-- >>> data V2 a = V2 a a deriving Functor
```

Comment:
Maybe

```suggestion
-- >>> data Pair a = Pair a a deriving Functor
```

Reply:
I really dislike writing code by clicking buttons in GitHub UI :S, but I like the suggestion.

Reply:
Embrace the squash merge!

Reply:
Anyway, in general I like `Pair` more then `V2`, but `Pair` is also a constructor of `Product f g`, which is very unfortunate in some cases. Here we don't use `Data.Functor.Product` for anything, so I pushed the change.